python/nano/src/bigdl/nano/deps/openvino/tf/model.py

```diff
@@ -79,3 +83,6 @@ def _load(path):
             invalidInputError(False, "nano_model_meta.yml must specify 'xml_path' for loading.")
         xml_path = Path(path) / status['xml_path']
         return KerasOpenVINOModel(xml_path)
+
+    def _save_model(self, path):
+        self.ov_model._save_model(path, model=self)
```

Comment:
what is this change for?

Reply:
`_save_model` in OpenVINOModel needs model's status property, but it can't be retrieved when OpenVINOModel is not the base class of PytorchOpenVINOModel and KerasOpenVINOModel, so I add a `model` parameter to `_save_model` in OpenVINOModel and pass the model object to `model` parameter.

Besides this change, we can also simply copy the implementation of `_save_model` in OpenVINOModel to PytorchOpenVINOModel and KerasOpenVINOModel, but I think it will be a little redundant.

Reply:
This is an issue we need to resolve. Is there any way to fix this without modifying the signature? We should have the same usage:
```python
OpenVINOModel.save()
PytorchOpenVINOModel.save()
KerasOpenVINOModel.save()
```
If you must retrieve status, just pass `self.status`.

Reply:
ok, I will change it.